src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,110 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getFilmByTitle } from '../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ onAddMovie }) => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [findError, setFindError] = useState(false);
+  const [film, setFilm] = useState(null);
+
+  const findMovie = async() => {
+    try {
+      const newFilm = await getFilmByTitle(searchQuery);
+
+      setFindError(false);
+      setFilm(newFilm);
+      setSearchQuery('');
+    } catch (error) {
+      setFindError(true);
+      setFilm(null);
+    }
```

Comment:
А как тебе такой вариант
```suggestion
    const newFilm = await getFilmByTitle(searchQuery)
      .catch(() => null)

    setFindError(!newFilm);
    setFilm(newFilm);
      
    if (newFilm) {  
      setSearchQuery('');
    }
```